components/ei-solutions/org.wso2.carbon.ei.solutions.ei.analytics/widgets/MediatorProperties/src/resources/widgetConf.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "MEDIATOR PROPERTIES",
+  "id": "MediatorProperties",
+  "thumbnailURL": "",
+  "configs": {
+    "pubsub": {
+      "types": [
+        "subscriber"
+      ]
+    },
+    "providerConfig": {
+      "configs": {
+        "type": "SiddhiStoreDataProvider",
+        "config": {
+          "siddhiApp": "@App:name('ESBEventTable') @store(type = 'rdbms', datasource = 'EI_ANALYTICS', field.length='contextPropertyMap:5000,beforePayload:5000,afterPayload:5000,transportPropertyMap:5000' ) @Index('metaTenantId','messageFlowId','host','hashCode','componentName','componentType','componentIndex','componentId','startTime','endTime','entryPoint','entryPointHashcode','faultCount') define table ESBEventTable (metaTenantId int, messageFlowId string, host string, hashCode string, componentName string, componentType string, componentIndex int, componentId string, startTime long, endTime long, duration long, beforePayload string, afterPayload string, contextPropertyMap string, transportPropertyMap string, children string, entryPoint string, entryPointHashcode string, faultCount int, eventTimestamp long);",
```

Comment:
Use the same siddhi app name.. for easy debugging later on. However, add only the definitions needed. (Event table/Aggregations) remove all processing logic

Reply:
Ignore the comment on definitions. seems only table definitions are their

Reply:
Fixed with ( 8d4a31838382c2521640a73fbada3caf4c73101b 7b2df2b095d7ce643a2b920943ee88cfbab0d3bc 96d518077e89317005fe59ff8670a05d4f705b4b 9ae3d822e7addb1c50a2d7603489a4dd64be3180 07bb8f914fefa6f383f6b7ffce467290a322e356 d40163630eec02a0bee35aec75ef149842b4f45e 0df301308a5496995274a95eed4b528881a5e484 )